monitor.js

```diff
@@ -222,6 +223,19 @@ const Monitor = module.exports = {
 		return count > 4;
 	},
 
+	/**
+	 * Counts ticket creation. Returns true if too much.
```

Comment:
Should ac accounts also be an exception to this?

Reply:
Possibly, though idk if a user could burn through 5 tickets in an hour (maybe, if so possibly just up the limit?)

Reply:
It's more relevant for shared IPs, I think.